homework/src/index.js

```diff
@@ -30,18 +29,74 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { html: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { html: JSON.stringify(data, null, 2) });
+  const HYF_REPOS_URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+  const root = document.getElementById('root');
+  createAndAppend("img", root, { src: "./hyf.png", id: "imgLogo" });
+  const divSelect = createAndAppend("div", root, { id: "divSelect" });
+  createAndAppend("p", divSelect, { html: "HYF Repositories : ", id: "p1" });
+  const select = createAndAppend("select", divSelect, { id: "select" });
+  const divFlex = createAndAppend("div", root, { id: "divFlex" });
+  const divInfo = createAndAppend("div", divFlex, { id: "divInfo" });
+  const divCont = createAndAppend("div", divFlex, { id: "divCont" });
+  const divError = createAndAppend("div", root, { id: "divError" });
```

Comment:
you have a lot of divs I wish you won't confuse too much in a huge application with a thousand line of code.

you can change names to be `container` or `header` or `section` `nav` `article` `content` `aside` you have a various kind of names and `header` or `section` `nav` `article` are a valid tags.

using described names is very helpful to remember what is each one and where it's exactly.

Aside of that you are creating the `divError` without any reason, if no error happen so you have an extra div in the page, in other cases greater apps you may have a 100 of them without any reason with a different kind of `id`s and variable names and more memory usage, in the previous case I said you may crash your page in sometime without any detecting from where the happen!!

Reply:
Corrected.